kstars/skycomponents/cometscomponent.cpp

```diff
@@ -184,6 +187,38 @@ void CometsComponent::loadData() {
         objectLists( SkyObject::COMET ).append(QPair<QString, const SkyObject*>(com->name(),com));
     }
 }
+/*
+void CometsComponent::draw( SkyPainter *skyp )
+{
+    Q_UNUSED(skyp)
+#ifndef KSTARS_LITE
+    if( !selected() || Options::zoomFactor() < 10*MINZOOM )
+        return;
+
+    bool hideLabels =  ! Options::showCometNames() ||
+                       (SkyMap::Instance()->isSlewing() &&
+                        Options::hideLabels() );
+    double rsunLabelLimit = Options::maxRadCometName();
+
+    //FIXME: Should these be config'able?
```

Comment:
Comments like these always scare me, like "Here be dragons." I have no idea what should be done, but I feel like it should be decided one way or the other rather than leaving this note for the future.